openbook_posts/models.py

```diff
@@ -210,7 +210,11 @@ def get_post_comment_reply_notification_target_users(cls, post_commenter, parent
 
         # Add post comment creator
         post_comment_creator = User.objects.filter(pk=parent_post_comment.commenter_id)
-        return other_repliers.union(post_comment_creator)
+
+        # Add post creator
+        post = parent_post_comment.post
```

Comment:
What is this for? Was something failed related to this? If so, can we write a test?

Reply:
Yep, this PR also fixes https://github.com/OkunaOrg/okuna-api/issues/489 (commented above)
This line adds the creator to the target users of replies, which we then check if present and use it to send a notification to them.